app/functions.R

```diff
@@ -50,6 +50,9 @@ formatNumber<-function(number, unit) {
 formatPercent<-function(probability) {
   return (formatNumber(100 * probability, "%"))
 }
+formatProbability<-function(probability) {
+  return (formatDynamicString(format(round(100/probability), big.mark=",")))
```

Comment:
I think this should be 1/probability, not 100, right?
I tested the same individual on the deployed app and the new feature:
deployed app: 
![image](https://user-images.githubusercontent.com/8843067/146847931-d84e5131-21c3-4e5d-a08d-e9cdd3e744ec.png)

new feature:
![image](https://user-images.githubusercontent.com/8843067/146847954-6d53721e-8f1f-4214-bed9-30683a213cd5.png)

Reply:
Thanks for catching! Will update in next push.